build.sbt

```diff
@@ -6,30 +6,29 @@ ThisBuild / scalaVersion := "2.13.11"
 
 lazy val root = (project in file("."))
   .settings(
-    name := "da-reference-generator"
+    name := "da-reference-generator",
+    libraryDependencies ++= Seq(
+      awsSdkDynamoDbV2,
+      awsSdkDynamoDbV1 % Test,
+      lambdaJavaCore,
+      lambdaJavaEvents,
+      scalaTest % Test,
+      ocitools,
+      circeCore,
+      circeGeneric,
+      circeParser,
+      scalaLogging,
+      logback,
+      logstash,
+      typesafe,
+      testContainer % Test,
+      testContainerDynalite % Test
+    )
   )
 
-libraryDependencies ++= Seq(
-  awsSdkDynamoDbV2,
-  awsSdkDynamoDbV1 % Test,
-  lambdaJavaCore,
-  lambdaJavaEvents,
-  scalaTest % Test,
-  ocitools,
-  circeCore,
-  circeGeneric,
-  circeParser,
-  scalaLogging,
-  logback,
-  logstash,
-  typesafe,
-  testContainer % Test,
-  testContainerDynalite % Test
-)
-
 (assembly / assemblyJarName) := "reference-generator.jar"
 
 (assembly / assemblyMergeStrategy) := {
-  case PathList("META-INF", xs @_*) => MergeStrategy.discard
+  case PathList("META-INF", "MANIFEST.MF") => MergeStrategy.discard
```

Comment:
So I'm guessing something was missing in the JAR that was needed for logging?

Reply:
I don't think it was the Jar I just copied this from the file-format lambda. I think the issue was the instantiating of the Logger. 😅